### DISCUSSION TITLE
Eloura74's Floorplan (Note: Not ha-floorplan, but done with picture_element)

### DISCUSSION THREAD

**Discussion by Eloura74** [ExperienceLovelace/ha-floorplan#418] [Show and tell]
**Presentation de mon Floorplan:** <br> <br>
![Floor1](https://github.com/ExperienceLovelace/ha-floorplan/assets/95535946/2a7fc531-a3fd-464e-b4b0-4dda76b18651)
![Floor2](https://github.com/ExperienceLovelace/ha-floorplan/assets/95535946/5ef6b877-77d6-450e-b2c3-34f8e91f15e2)
![Floor3](https://github.com/ExperienceLovelace/ha-floorplan/assets/95535946/6f76106f-728c-48c6-a13d-fa2614c5b668)
![Floor4](https://github.com/ExperienceLovelace/ha-floorplan/assets/95535946/90c4c04f-ded0-4a29-b158-528d326b71ed)

https://github.com/ExperienceLovelace/ha-floorplan/assets/95535946/b7542fd9-f8d7-449d-b017-3d4e2375eb5c

https://github.com/ExperienceLovelace/ha-floorplan/assets/95535946/fb1e541e-d3d0-4dcf-8557-b01a9c0233c7

Voici un petit aperçu de mon floor plan, <br>
J'ai créer les plan de ma maison avec sweethome3d, ensuite travaillé mes images avec Gimp. <br>
puis j'ai fait mon floor plan avec une picture-element: <br>
[Floor_plan.zip](https://github.com/ExperienceLovelace/ha-floorplan/files/14096419/Floor_plan.zip)

<br>
j'ai utilisé quelques animations en css, des images personalisées pour les honeycomb, la presence de personne, le fond d'image, etc <br>
J'ai également passé du temps à générer des images de chaques éclairages, chaques portes ouvertes, à différentes heures de la journées pour avoir un rendu de luminositée différent en fonction du moment dans la journée

**Comment by OzGav** [comment]
Really nice but you have used picture elements and have posted in the repo for the ha-floorplan custom component!

**Reply by Trzinka** [reply]
# I asked him/her to share his/her project because it is one of the most beautiful. 

**From the attached files in the zip file, it will be surely possible to get some idea.**

> `Perhaps another important thing: from the beginning I wondered whether it is better to use`
> - _picture elements card_
> or
> - _floor plan card_
> `(and still wondering and would like to see the difference in code etc.)`

# After all, both show the floor plan in the HA front end!

### Of course, I could be wrong.

**Reply by ramdisk00** [reply]
for sure one of the best

**Reply by Tamooj** [reply]
Consider it a set of feature requests :-)

**Reply by exetico** [reply]
@Tamooj ; What kind of features are you missing? :)

**Comment by Trzinka** [comment]
# My personal opinion is that this is one of the most beautiful floor plans.

## Thanks for sharing the files!!

**Reply by Eloura74** [reply]
I hope this can help you

**Reply by Eloura74** [reply]
I'm trying to do a more detailed markdown, but I don't have time. :D

**Reply by Trzinka** [reply]
## Don't worry, if someone doesn't understand, they'll probably ask!!!!

**Comment by Trzinka** [comment]
**Hi.** 
# I would like to ask you if you use this same yaml also for display on a mobile phone, because I am completely confused with the display on a mobile phone. 

Prime discussions: https://github.com/ExperienceLovelace/ha-floorplan/discussions/419.

or:

https://www.facebook.com/groups/HomeAssistant/?multi_permalinks=3689608521310491&notif_id=1708940614597274&notif_t=feedback_reaction_generic&ref=notif

Do you have any idea how to solve this?

**Reply by OzGav** [reply]
Yes same yaml different image.

**Comment by exetico** [comment]
Great work There's a impressive amount of different images, too. Is it made with Sweet Home 3D? It must have been a pain to cut all whose images out, and get the best (as possible) result for each render/cutout?

With that said, I can't see where you involved **ha-floorplan**, which are the area where you've shared your work? :)

Update: Oh, I just saw the answer from [Trzinka](https://github.com/Trzinka). Now it makes sense.
